Directory.Packages.props

```diff
@@ -18,9 +18,9 @@
     <MicrosoftBuildVersion Condition="'$(TargetFramework)' == 'netcoreapp3.1' Or '$(TargetFramework)' == 'netstandard2.0'">16.8.0</MicrosoftBuildVersion>
     <!-- The last package version of MSBuild that works with net5.0 is 16.11.0.  Our assemblies are still compiled against MSBuild assembly version 15.1.0.0 which will work with all versions -->
     <MicrosoftBuildVersion Condition="'$(TargetFramework)' == 'netcoreapp5.0'">16.11.0</MicrosoftBuildVersion>
-    <MicrosoftBuildVersion Condition="'$(DotNetBuildSourceOnly)' == 'true'">17.12.6</MicrosoftBuildVersion>
+    <MicrosoftBuildVersion Condition="'$(DotNetBuildSourceOnly)' == 'true'">17.13.9</MicrosoftBuildVersion>
     <!-- Read docs/updating-packages.md before updating MSBuild's version -->
-    <MicrosoftBuildVersion Condition="'$(MicrosoftBuildVersion)' == ''">17.11.4</MicrosoftBuildVersion>
+    <MicrosoftBuildVersion Condition="'$(MicrosoftBuildVersion)' == ''">17.13.9</MicrosoftBuildVersion>
```

Comment:
The comment says to read docs/updating-packages.md, and that doc explains why we can't update the package.

If you find the docs confusing, please talk to me and then update the docs to be easier to understand, but we can't take this change.

Reply:
if it helps, we can have `src/NuGet.Clients/**/*.csproj` use MSBuild 17.13, although I'm not sure if any of those projects actually reference MSbuild.  It's `src/NuGet.Core/**/*.csproj` that need to say on 17.12

Reply:
I can't remember why I updated it, lets try without it...

Reply:
Lots of downgrade warnings, I'll try suppressing them.